src/main/resources/data/spleef/games/standard_projectile.json

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "spleef:spleef",
+  "map": {
+    "shape": {
+      "type": "spleef:circle",
+      "radius": 16
+    },
+    "levels": 4,
+    "level_height": 5
+  },
+  "projectile": {
+    "enabled": true
```

Comment:
hmm, would it make sense to have the projectile config be enabled based on whether the projectile field is declared or not? it might be a bit weird to have just like a `"projectile": {}` so I'm not totally sure

Reply:
Yes, that would be better, but I'm not sure whether codecs support that.

Reply:
They *should*, an empty json object should still be treated as existing. Can use `optionalFieldOf` and store the projectile config as an optional

Reply:
Fixed in 59ad12ff5313c3b0b00ab492ac25dc3ba468f198